internal/providers/terraform/azure/testdata/storage_account_test/storage_account_test.golden

```diff
@@ -1,91 +1,139 @@
- ├─ Data write                                             7,000  GB                              $35.00 
- └─ Blob index                                                70  10K tags                         $4.83 
-                                                                                                         
- azurerm_storage_account.Standard_RAGRS_Hot                                                              
- ├─ Capacity (first 50TB)                                 51,200  GB                           $3,015.68 
- ├─ Capacity (next 450TB)                                512,000  GB                          $28,950.53 
- ├─ Capacity (over 500TB)                              7,436,800  GB                         $402,985.32 
- ├─ Write operations                                         800  10K operations                  $80.00 
- ├─ List and create container operations                     800  10K operations                  $80.00 
- ├─ Read operations                                           80  10K operations                   $0.32 
- ├─ All other operations                                     800  10K operations                   $3.20 
- └─ Blob index                                                80  10K tags                         $5.52 
-                                                                                                         
- azurerm_storage_account.without_usage_file                                                              
- ├─ Capacity                                  Monthly cost depends on usage: $0.04 per GB                
- ├─ Write operations                          Monthly cost depends on usage: $0.20 per 10K operations    
- ├─ List and create container operations      Monthly cost depends on usage: $0.10 per 10K operations    
- ├─ Read operations                           Monthly cost depends on usage: $0.01 per 10K operations    
- ├─ All other operations                      Monthly cost depends on usage: $0.004 per 10K operations   
- ├─ Data retrieval                            Monthly cost depends on usage: $0.01 per GB                
- ├─ Data write                                Monthly cost depends on usage: $0.005 per GB               
- └─ Blob index                                Monthly cost depends on usage: $0.07 per 10K tags          
-                                                                                                         
- PROJECT TOTAL                                                                             $1,212,492.56 
-
+ Name                                                 Monthly Qty  Unit                Monthly Cost 
+                                                                                                    
+ azurerm_storage_account.bb_Premium_LRS                                                             
+ ├─ Capacity (first 50TB)                                  51,200  GB-months              $7,680.00 
```

Comment:
I think there is an issue here because storage_gb is set to 1000 TB in the usage file but we're only reporting costs for the first 50TB